plugins/hls-semantic-tokens-plugin/test/Main.hs

```diff
@@ -194,11 +193,9 @@ semanticTokensTests =
       goldenWithSemanticTokensWithDefaultConfig "type family" "TTypefamily",
       goldenWithSemanticTokensWithDefaultConfig "TUnicodeSyntax" "TUnicodeSyntax",
       goldenWithSemanticTokensWithDefaultConfig "TQualifiedName" "TQualifiedName"
-      -- it is not supported in ghc92
-#if MIN_VERSION_ghc(9,4,0)
-      , goldenWithSemanticTokensWithDefaultConfig "TDoc" "TDoc"
-#endif
     ]
+      -- not supported in ghc92
```

Comment:
CC @soulomoon I used this value-level CPP-less way to ensure this test only runs with GHC 9.4 and newer.
Did this to be able to get rid of "unused ghc dependency" warning.

Reply:
Thanx, it looks good